packages/google_sign_in/lib/google_sign_in.dart

```diff
@@ -127,6 +129,9 @@ class GoogleSignIn {
   static const MethodChannel channel =
       const MethodChannel('plugins.flutter.io/google_sign_in');
 
+  /// The options to pass to GoogleSignInOptions.Builder()
```

Comment:
We should not mention the Android API's builder here, but instead document what `SignInOption` is, in its own right. And we should document that `gamesSignIn` isn't supported on iOS.